### PR TITLE
Plotting the prediction intervals

### PR DESCRIPTION
**Pull request by Ifeanyi30**
#### Reference Issues/PRs

fixes the issue #541

#### What does this implement/fix? Explain your changes.

Added the pred_int as an option to the plot_series to plot the confidence interval of the predictions.
#### Does your contribution introduce a new dependency? If yes, which one?
No

#### What should a reviewer concentrate their feedback on?
Focus on the plot_series function in the plotting.py file.

The changes on the __init__ of the EnsembleForecaster and it predict method.
##### For all contributions
- [ ] I've added myself to the [list of contributors](https://github.com/alan-turing-institute/sktime/blob/main/.all-contributorsrc).
- [ ] Optionally, I've updated sktime's [CODEOWNERS](https://github.com/alan-turing-institute/sktime/blob/main/CODEOWNERS) to receive notifications about future changes to these files.
- [ ] I've added unit tests and made sure they pass locally.

##### For new estimators
- [ ] I've added the estimator to the online documentation.
- [ ] I've updated the existing example notebooks or provided a new one to showcase how my estimator works.

### CODE REVIEW

**Comment by mloning** [comment]
Hi @Ifeanyi30, I think the idea is rather to allow users to pass prediction intervals obtained from a forecaster, see for example the PR https://github.com/alan-turing-institute/sktime/pull/545

**Comment by mloning** [comment]
@Ifeanyi30 take a look at the tutorial notebook here: https://www.sktime.org/en/latest/examples/01_forecasting.html#Prediction-intervals - the prediction intervals already give upper and lower limits, so we only have to fill in the space between

**Comment by Ifeanyi30** [comment]
Thanks, the information was useful and I have made changes to the PR

**Comment by Ifeanyi30** [comment]
Hi @mloning, I have made the changes to the PR. Please review the changes.

**Comment by aiwalter** [comment]
@Ifeanyi30 would be nice if you could add this to the example notebook? :) So basically replacing the plot example there with this new functionality.
___
Martin Walter <martin_friedrich.walter@daimler.com>, Mercedes-Benz AG on behalf of Daimler TSS GmbH.
https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md

File: sktime/utils/plotting.py [hunk 1]
```diff
@@ -91,6 +93,25 @@ def plot_series(*series, labels=None, markers=None):
...
+                ax.get_lines()[-1].get_xdata(),
+                pred_int.lower,
+                pred_int.upper,
+                alpha=0.3,
+                color=ax.get_lines()[-1].get_c(),
+                label="prediction intervals",
+            )
+        else:
+            raise ValueError(
+                "pred_int has index values for which no other index values supplied"
```
Comment (mloning) [inline sktime/utils/plotting.py:112]: Could we also improve the error message, I guess we want to say that pred_int index does not match the index of any of the passed series.

**Comment by Ifeanyi30** [comment]
Hi @mloning, I have made changes to the error message and would like you to
review the code again.
In the meeting, you emphasized on the importance of making a sensible
contribution. Well, including this PR, I have made two contributions to
sktime, is it ok to record them for my final application?

**Comment by Ifeanyi30** [comment]
Hi Martin, so how do I do that?

**Comment by mloning** [comment]
@Ifeanyi30 regarding your application, your PRs are fine - thanks!

**Review by mloning — COMMENTED** [review]
Thanks again @Ifeanyi30 - looks all good to me so far. 

I agree with @aiwalter, please adjust the corresponding code in `examples/01_forecasting.ipynb`.

**Comment by mloning** [comment]
@Ifeanyi30 quick warning: before changing the notebook, please wait for PR https://github.com/alan-turing-institute/sktime/pull/793 to be merged, otherwise we will run into merge conflicts.

**Comment by Ifeanyi30** [comment]
Ok

**Comment by mloning** [comment]
@Ifeanyi30 #793 is now merged!

**Comment by Ifeanyi30** [comment]
Ok, thanks

**Comment by Ifeanyi30** [comment]
Hi guys, I used a windows operating system, when I commit and push the
examples/01_forevasting.ipynb file, Git automatically converts the Unix LF
format to windows' CRLF. I have tried configuring the git core.autocrlf to
false but still doesn't work and it tempers with the code quality.
What should I do?

**Comment by mloning** [comment]
@aiwalter can you help with that? I'm not on Windows. `git core.autocrlf` seems like the right place to change that.

**Comment by aiwalter** [comment]
@Ifeanyi30 I think you can ignore this warning, I also had it sometimes. Or what is the issue?

**Comment by Ifeanyi30** [comment]
When I push the changes the code quality test fails because of the new line
in the Unix format that is not present.

**Comment by aiwalter** [comment]
@Ifeanyi30 did you install and use pre-commit?

**Comment by Ifeanyi30** [comment]
Yes, I did.

**Comment by Ifeanyi30** [comment]
I suggest we merge this PR, then I keep trying to update the
examples/forecasting.ipynb file.

**Comment by mloning** [comment]
I'd prefer to update the notebook first as it also is a test for the new functionality. 

There seem to be a lot of discussions on the topic, see e.g. [here](https://stackoverflow.com/questions/15223290/fixing-mac-windows-new-lines-with-git-pre-commit-hook). Could you try to solve it again and perhaps post the (error) message that you receive?

**Comment by Ifeanyi30** [comment]
The error the message I got is attached below. It is clearly a platform
issue. I used 'git config --global core.autocrlf false' but it did not work.

**Comment by Ifeanyi30** [comment]
Maybe If I try resetting the sktime locally repo and git file format on my
PC then it might just work.

**Comment by mloning** [comment]
@Ifeanyi30 yes please see if you can solve it, otherwise post the error message so that we can help, or push the code to see why the CI checks are failing.

**Comment by Ifeanyi30** [comment]
This is the error code

**Comment by mloning** [comment]
@Ifeanyi30 let me know if I can help with anything!

**Comment by Ifeanyi30** [comment]
Thanks!

**Comment by Ifeanyi30** [comment]
This is the error message

**Comment by mloning** [comment]
@Ifeanyi30 sorry I can't see the message

**Comment by mloning** [comment]
Closed in favour of #802

**Comment by Ifeanyi30** [comment]
Yeah

**Pull request closed without merging**